pkg/audit/verifier_test.go

```diff
@@ -103,3 +108,60 @@ func TestNotEnoughShares(t *testing.T) {
 	_, _, err = auditShares(ctx, 20, 40, auditPkgShares)
 	require.Contains(t, err.Error(), "infectious: must specify at least the number of required shares")
 }
+
+func TestCreatePendingAudits(t *testing.T) {
+	const (
+		required = 8
+		total    = 14
+	)
+
+	f, err := infectious.NewFEC(required, total)
+	if err != nil {
+		panic(err)
```

Comment:
why a panic and not t.Fail or require.NoError?

Reply:
Because I blindly copied this part from other tests in the same file :)

I fixed it.